chart/templates/_helpers.yaml

```diff
@@ -380,7 +380,7 @@ server_tls_key_file = /etc/pgbouncer/server.key
         directory = os.path.join(package_dir, 'migrations')
         config = Config(os.path.join(package_dir, 'alembic.ini'))
         config.set_main_option('script_location', directory)
-        config.set_main_option('sqlalchemy.url', settings.SQL_ALCHEMY_CONN)
+        config.set_main_option('sqlalchemy.url', settings.SQL_ALCHEMY_CONN.replace('%', '%%'))
```

Comment:
Isn't that a problem of the wrong URL specified?

For the SQL alchemy connection should be URL-encoded in the first place. This is true for the individual parameters of the query but also for user and password. So I think your problem is that in your original URL you should us %25 where you have '%'  and not replace % with %%.

Reply:
The issue is I think that '%' in ConfigParser is special -- it's the same case as I fixed a few years ago in https://github.com/apache/airflow/pull/3787.